Lib/test/test_threading.py

```diff
@@ -915,6 +915,15 @@ def noop(): pass
             threading.Thread(target=noop).start()
             # Thread.join() is not called
 
+    def test_debug_deprecation(self):
```

Comment:
Tests are currently failing on Windows because those tests use release build.
```
2021-07-08T10:43:29.6477759Z ======================================================================
2021-07-08T10:43:29.6478377Z FAIL: test_debug_deprecation (test.test_threading.ThreadTests)
2021-07-08T10:43:29.6479112Z ----------------------------------------------------------------------
2021-07-08T10:43:29.6479861Z Traceback (most recent call last):
2021-07-08T10:43:29.6480603Z   File "D:\a\cpython\cpython\lib\test\test_threading.py", line 925, in test_debug_deprecation
2021-07-08T10:43:29.6481279Z     self.assertIn(msg, err)
2021-07-08T10:43:29.6481772Z     ^^^^^^^^^^^^^^^^^^^^^^^
2021-07-08T10:43:29.6482882Z AssertionError: b'DeprecationWarning: The threading debug (PYTHONTHREADDEBUG environment variable) is deprecated and will be removed in Python 3.12' not found in b''
2021-07-08T10:43:29.6483939Z 
2021-07-08T10:43:29.6484468Z ----------------------------------------------------------------------
```
I think you should add a check for if the current CPython is indeed in debug build. I'm not sure what's the best/safest way to do that, so I'll leave that up to you :).

Reply:
the latter hasattr probably works as that's an API only present in pydebug builds IIRC.

Reply:
Fixed, thanks.